Source/WebCore/page/Quirks.cpp

```diff
@@ -1554,4 +1554,9 @@ bool Quirks::shouldEnableApplicationCacheQuirk() const
 #endif
 }
 
+bool Quirks::shouldEnableFontLoadingAPIQuirk() const
+{
+    return true;
```

Comment:
That's it? We're not checking for a domain or anything?

Seems like at the very least we should check for `needsQuirks()`

Reply:
I marked this PR as a draft